initramfs/src/image.rs

```diff
@@ -126,30 +130,35 @@ impl Image {
             File::open(init_path).map_err(|e| anyhow!(e).context("Failed to open init file"))?;
         let mut agent_file =
             File::open(agent_path).map_err(|e| anyhow!(e).context("Failed to open init file"))?;
+        let mut agent_config_file = File::open(agent_config_path)
+            .map_err(|e| anyhow!(e).context("Failed to open agent config file"))?;
 
         let mut init_content = Vec::new();
         let mut agent_content = Vec::new();
+        let mut agent_config_content = Vec::new();
 
         init_file.read_to_end(&mut init_content)?;
         agent_file.read_to_end(&mut agent_content)?;
-
-        let init_mode = init_file.metadata()?.permissions().mode();
-        let agent_mode = agent_file.metadata()?.permissions().mode();
+        agent_config_file.read_to_end(&mut agent_config_content)?;
 
         entries.insert(
             "init".to_string(),
-            (
-                Builder::new("init").mode(init_mode),
-                Cursor::new(init_content),
-            ),
+            (Builder::new("init").mode(33277), Cursor::new(init_content)),
         );
         entries.insert(
             "agent".to_string(),
             (
-                Builder::new("agent").mode(agent_mode),
+                Builder::new("agent").mode(33277),
                 Cursor::new(agent_content),
             ),
         );
+        entries.insert(
+            "config.yaml".to_string(),
+            (
+                Builder::new("config.yaml").mode(33204),
```

Comment:
Same for 33204. You should define a constant for those, with a relevant name.

Reply:
Same here. I'll create a constant